daemon/cluster/executor/container/container.go

```diff
@@ -139,6 +139,11 @@ func (c *containerConfig) labels() map[string]string {
 		labels[k] = v
 	}
 
+	// we apply service level labels, which could be obtained from ServiceAnnotations.Labels
+	for k, v := range c.task.ServiceAnnotations.Labels {
+		labels[k] = v
+	}
```

Comment:
There has been a lot of discussion about  this, and so far the agreement is that we don't want to copy the labels... yet. It needs more discussion apparently.

So instead of copying the label, we can apply a container label from the client , in the same place we add the service label.